cmd/avalanche.go

```diff
@@ -23,6 +44,7 @@ var (
 	valueInterval       = kingpin.Flag("value-interval", "Change series values every {interval} seconds.").Default("30").Int()
 	labelInterval       = kingpin.Flag("series-interval", "Change series_id label values every {interval} seconds.").Default("60").Int()
 	metricInterval      = kingpin.Flag("metric-interval", "Change __name__ label values every {interval} seconds.").Default("120").Int()
+	ignoredTimers       = IgnoredIntervals(kingpin.Flag("ignored-intervals", "Comma-separated list of intervals which should not be run"))
```

Comment:
The flexibility of toggling specific intervals on/off is appreciated, but seems a little confusing how this should be used based on the help message.  Thoughts on using a single bool flag to toggle both `label` and `metric` changing off?  That seems more what the initial request in https://github.com/prometheus-community/avalanche/issues/27 is.